packages/http-helper/src/mod.ts

```diff
@@ -4,6 +4,7 @@ export * as ClearSiteData from './clear_site_data.js';
 export * as CSP from './csp/mod.js';
 export * as HttpHeaderName from './http_header_name.js';
 export * as HttpMethod from './http_method.js';
+export type { HttpMethods } from "./http_method.js"
```

Comment:
1. I think this might be better to define by `...keyof (typeof HttpMethod)`. Its pattern will never forget to add new item to it type.
2. We don't expose `http_method.js` via subpath exports. We can define its type at here.